labshare/tests.py

```diff
@@ -463,6 +492,147 @@ def test_reservation_str_representation(self):
         ))
 
 
+class TestReservationExpiration(LabshareTestSetup):
+    @classmethod
+    def setUpTestData(cls):
+        LabshareTestSetup.setUpTestData()
```

Comment:
`super().setUpTestdata()`? Does that work on a class method?

Reply:
No

Reply:
At least I think so. :D

Reply:
it works, I just checked it^^